distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -64,21 +83,20 @@ Non-Terminating authentication types are authentication types where, once hit, m
 If the client supports the non-terminating authentication type's method of obtaining credentials, it either allows or forbids access to the system.
 However if the client does not support the non-terminating authentication type's method of obtaining credentials, the system will continue to the next configured authentication type.
 
-`BASIC` is the only terminating authentication type.
-Every other authentication type is non-terminating.
+`PKI` is the only non-terminating authentication type.
+`BASIC`, `OIDC`, and `SAML` are terminating authentication types.
 
-For example: assume a context is protected by the non-terminating `SAML` authorization type.
+For example: assume ${branding} is protected by the terminating `SAML` authorization type.
 The system first checks to see if the client supports the acquisition of SAML credentials.
 
 - If the connecting client is a browser, the system can acquire SAML credentials.
 - If the connecting client is a machine that supports SAML ECP, the system can acquire SAML credentials.
 - If the connecting client is a machine that does not support SAML ECP, the system cannot acquire SAML credentials.
 
 If the system can acquire SAML credentials from the client, the system will attempt to acquire said credentials and either allow or forbid access.
-If the system cannot acquire SAML credentials from the client, the system will continue to the next configured authentication type.
+If the system cannot acquire SAML credentials from the client, the system will forbid access.
 
-Contrarily, assume a context is protected by the terminating `BASIC` authentication type.
-Once this authentication type is hit, the system either allows or forbids access to the system, without checking if the client supports the acquisition of BASIC credentials.
+Contrarily, if ${branding} is protected by the non-terminating `PKI` authorization type and the system cannot acquire PKI certificates from the client, the system will continue to the next configured authentication type.
```

Comment:
✏️ Minor nit-pick to be sure, but "contrarily" ? Could we maybe say "Otherwise" instead?